new folder to hold all icons

Gayle-Thompson-Igwebike opened the pull request:
#58-All-Icons-Component-Gayle-Thompson-Igwebike

## Description

Created a new folder to hold all the icons for the teams page. Plus simple basic css. -  It probably wont show just now, because the other components need to be merged first.

## Related to #58 


Fixes #58 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [x] I have commented my code
- [ ] I have updated any documentation

Gayle-Thompson-Igwebike committed:
new folder to hold all icons

Plus simple basic css.

AndrianaOS commented on client/src/components/Icons-Components/All-IconsContainer.jsx:
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './All-IconsContainer.css';
+import SlackIconSvg from './SlackIconSvg';
```
Your paths dont point into any folders. They need to point to the file containing the images

Gayle-Thompson-Igwebike replied:
OK. Thank you.

AndrianaOS reviewed (dismissed):
Please configure your paths. They crash the page when included in App.js as they dont have the right file path

Gayle-Thompson-Igwebike committed:
Functional Icons Container

Updated links to each icon- updated names of components. Moved files and recreated accurate file paths to render on browser. Icons container is now functional. Minimum css styling added. Can update if needed.

Gayle-Thompson-Igwebike committed:
updated slack icon

Gayle-Thompson-Igwebike commented:
The file paths have been resolved

Gayle-Thompson-Igwebike committed:
tidy app.js

Gayle-Thompson-Igwebike committed:
resolve file conflicts

Gayle-Thompson-Igwebike commented on client/src/components/Icons/DeployedAppIconSvg.jsx:
Added correct links for each icon but I think they are causing conflicts so changed them back

Gayle-Thompson-Igwebike committed:
Merge branch 'dev' into 58-all-icons-component

Gayle-Thompson-Igwebike committed:
update slackicon

Gayle-Thompson-Igwebike committed:
slaxk update
